lib/challenge.js

```diff
@@ -61,6 +91,24 @@ var some_pet;
 var Pet;
 // Answer Starts Here
 
+var Pet = function Pet(name, species, breed, noise) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = []];
+
+};
+
+Pet.prototype.makeNoise = function makeNoise () {
+  var makeNoises = 0;
+  for (var i = 0; i < this.noise.length; i++) {
+    makeNoises += this.noise[i].noise;
+  }
+  return makeNoise;
+};
```

Comment:
So, given that you chose to take the approach of iterating over an array of noises, and concatenating strings stored in the elements of that array, the initial value of your accumulator (the variable you're combining everything in) is contrary to your desires.

You set the initial value of `makeNoises` to `0`, a number. When you add a number to a string, the number gets turned into a string to make sense of it. As a result, your string of noises would start with `'0'`, e.g., `'0woofwoof'`.

A good initial value for a string accumulator is the empty string, `''`. It's like the zero of strings. If you concatenate it with another string, you get the other string every time, just like how if you add zero to another number, you get the other number every time.